src/components/PageHeader/PageHeader.style.ts

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+export const StyledPageHeader = styled.header<{ size: 'small' | 'large' }>`
+    .header-container {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        width: 100%;
+        max-width: ${props => (props.size === 'small' ? '600px' : '1200px')};
+        height: ${props => (props.size === 'small' ? '60px' : '100px')};
+        padding: ${props => (props.size === 'small' ? '0.5rem' : '1rem')};
+        margin: 0 auto;
+        background-color: #333; /* Background color for the header */
+        box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1); /* Optional shadow for header */
+    }
+
+    .logo-container {
+        flex-shrink: 0;
+    }
+
+    .logo {
+        height: 100%;
+        max-height: ${props => (props.size === 'small' ? '50px' : '80px')}; /* Adjust logo size */
+        width: auto;
+    }
+
+    .menu-container {
+        display: flex;
+        gap: ${props => (props.size === 'small' ? '0.5rem' : '1rem')}; /* Space between menu items */
+    }
+
+    .menu-item {
+        all: unset;
+        color: white; /* Text color for menu items */
```

Comment:
Please use the theme colours here, because different projects may use different colour schemes and white might not provide sufficient contrast. Consider using Polished's `readableColor` function to automatically choose white/black based on the background colour.